src/site/content/en/react/nextjs-route-prefetching/index.md

```diff
@@ -0,0 +1,187 @@
+---
+layout: post
+title: Route prefetching in Next.js
+authors:
+  - mihajlija
+subhead: |
+  How Next.js speeds up navigations with route prefetching and how to customize it.
+date: 2019-10-22
+draft: true
+---
+
+## The `<Link>` component
+
+In [Next.js](https://nextjs.org/), you don't need to set up routing manually.
+Next.js uses file-system-based routing, which lets you just create files and folders
+inside the `./pages/` directory:
+
+<figure class="w-figure--center">
+<img class="w-screenshot" src="./pages-directory.png" alt="Screenshot of the pages directory containting three files: index.js, margherita.js, and pineapple-pizza.js.">
+</figure>
+
+To link to different pages, use the
+[`<Link>`](https://nextjs.org/docs#with-link) component, similarly to how you'd
+use the good old `<a>` element:
+
+```js
+<Link href="/margherita">
+  <a>Margherita</a>
+</Link>
+```
+
+When you use the `<Link>` component for navigation, Next.js does a little bit
+more for you. Normally, a page is downloaded when you follow a link to it, but
+[Next.js automatically prefetches](https://nextjs.org/docs#prefetching-pages)
+the JavaScript needed to render the page.
+
+When you load a page with a few links, odds are that by the time you follow
+a link, the component behind it has already been fetched. This improves
+application responsiveness by making navigations to new pages quicker.
+
+In the example app below, the `index.js` page links to `margherita.js` with a
+`<Link>`:
+
+<div class="glitch-embed-wrap" style="height: 480px; width: 100%;">
+<iframe src="https://glitch.com/embed/#!/embed/nextjs-prefetching?path=pages/index.js&attributionHidden=true"
+        alt="A basic Next.js app on Glitch"
+        style="height: 100%; width: 100%; border: 0;"></iframe>
+</div>
+
+Use Chrome DevTools to verify that `margherita.js` is prefetched:
+{% Instruction 'preview', 'ol' %}
+
+{% Instruction 'devtools-network', 'ol' %}
+
+{% Instruction 'disable-cache', 'ol' %}
+
+{% Instruction 'reload-page', 'ol' %}
+
+When you load `index.js`, the **Network** tab shows that `margherita.js` is
+downloaded too:
+
+![Screenshot of DevTools Network tab with margherita.js highlighted.](network1.png)
```

Comment:
I think you can remove "Screenshot of" from your alt text because the user will already know this is an image. I'd suggest doing this throughout.